src/depackers/stb_vorbis.c

```diff
@@ -741,12 +1121,14 @@ static int compute_codewords(Codebook *c, uint8 *len, int n, uint32 *values)
       }
 
       while (z > 0 && !available[z]) --z;
-      if (z == 0) { /*assert(0);*/ return FALSE; }
+      if (z == 0) { return FALSE; }
       res = available[z];
+      assert(z >= 0 && z < 32);
```

Comment:
This assert is wrong: at the very least it be a line above.
Also, why not return failure and assert() instead?

Reply:
The better question is why this assert exists at all: both failure cases are impossible. Line 1119 prevents the >=32 case and lines 1123+1124 prevent the <0 case.